R/class-AbundanceDensityPlot.R

```diff
@@ -293,6 +327,11 @@ setMethod(".definePanelTour", "AbundanceDensityPlot", function(x) {
                 paste0(panel_name, "_dots_colour"), "Column data",
                 iSEE:::.selectInputHidden(x, field="dots_colour_by",
                     label="Color dots by", choices=names(colData(se)), 
-                    selected=slot(x, "dots_colour_by"))))
+                    selected=slot(x, "dots_colour_by"))),
+            .conditionalOnRadio(
```

Comment:
Maybe insteading of making it depend on layout, you can just use the same approach as for "dots_colour". Later on we might change this into the visual option checkbox approach

Reply:
Could you add the parameter `add_legend` to the AbundanceDensityPlot panel? RowTreePlot can be used as a reference.